src/borg/testsuite/crypto.py

```diff
@@ -342,22 +342,25 @@ def test_decrypt_key_file_v2_is_unsupported(monkeypatch):
         key.decrypt_key_file(encrypted, passphrase)
 
 
-@pytest.mark.parametrize('key_algorithm', ('argon2 aes256-ctr hmac-sha256', 'sha256'))
-def test_key_file_roundtrip(monkeypatch, key_algorithm):
+@pytest.mark.parametrize('CLI_argument, expect_algorithm', (
+    ('argon2', b'argon2 aes256-ctr hmac-sha256'),
+    ('pbkdf2', b'sha256'),
```

Comment:
can you use `KEY_ALGORITHMS.items()`?

Reply:
or even without `.items()`?

Reply:
and maybe keep the uppercase names for classes, `cli_argument` is fine, too.

Reply:
`expected_algorithm` reads a bit nicer.

Reply:
> or even without `.items()`?

It does not work without `.items()` - I need both keys and values